README.md

```diff
@@ -49,34 +49,34 @@ Configuration is handled by environment variables. The shell scripts will show a
 
 To run just the scenarios locally, follow these steps:
 
-1. Install Vagrant from [vagrantup.com](https://vagrantup.com)
+1. Install Vagrant from [vagrantup.com](https://vagrantup.com). (Note: version after 2.2.3 have altered WinRM upload behaviour which may cause issues)
```

Comment:
Lets:
* add an issue to this repo about this
* add a version check to the startup script to stop other people finding this the hard way
* add an issue to the vagrant repo about this

Reply:
Issue added here